src/util/source_location.h

```diff
@@ -185,6 +185,9 @@ class source_locationt:public irept
     return static_cast<const source_locationt &>(get_nil_irep());
   }
 
+  /// get a path to the file, including working directory
```

Comment:
This is is different comment string from what's placed with the implementation, I'm not sure what Doxygen is going to do about this. The guidelines say that we only place documentation with the implementation.

Reply:
done. Not sure I agree with the guidelines. People read the class.

Reply:
Ack - how about submitting a PR that proposes a change `CODING_GUIDELINES.md`? I'd say that's the way to instigate change.